internal/attribute/attribute_test.go

```diff
@@ -94,3 +94,9 @@ func TestSliceValue(t *testing.T) {
 		})
 	}
 }
+
+func BenchmarkSliceValue(b *testing.B) {
```

Comment:
Can you please add benchmarks for all modified functions?

Reply:
Yes